### DISCUSSION TITLE
Trouble with Bay TV App and M3U Streaming: Need Help

### DISCUSSION THREAD

**Discussion by AssanSav** [iptv-org/community#1031] [Q&A]
Hello, I'm new to M3U streaming. I activated the Bay TV app on my smart TV, paid for it, and used the provided link (https://iptv-org.github.io/iptv/index.m3u) with my Bay TV app Mac code. However, I can only see the live category TV channels, and when I click on them, it gets stuck in download mode. Any suggestions?

![20240225_175243](https://github.com/iptv-org/iptv-org.github.io/assets/50751987/32bb062c-6733-43b1-bc08-a363f6b427e0)

**Comment by BellezaEmporium** [comment]
I would think that it's due to the massive amount of channels that are in this category.

Best-fitting choice for you would be the "index.country.m3u" playlist, available at https://iptv-org.github.io/iptv/index.country.m3u

**Reply by AssanSav** [reply]
Thanks for your suggestions, but unfortunately this playlist shows an empty list

**Reply by freearhey** [reply]
Then you should probably contact the developers of the Bay TV app.
